embrace-android-sdk/src/main/java/io/embrace/android/embracesdk/payload/PerformanceInfo.kt

```diff
@@ -74,5 +75,11 @@ internal data class PerformanceInfo(
      * StrictMode violations captured during the session
      */
     @SerializedName("v")
-    val strictmodeViolations: List<StrictModeViolation>? = null
+    val strictmodeViolations: List<StrictModeViolation>? = null,
+
+    /**
+     * Stats about the responsiveness of the ANR monitoring components
+     */
+    @SerializedName("rms")
```

Comment:
Not sure if this is the best place for this since it'll always update, essentially busting the cache all the time. Is there something else that updates that frequently in this payload that makes this irrelevant? If not, I should probably make it parallel to `p` and just add the plumbing to do that?

Reply:
`p` has its cache busted by `networkInterfaceIntervals` already so this seems a good place to put it